symbols.xml

```diff
@@ -1220,8 +1224,8 @@
         <global-address name='process_dig' value='0x141026a03'/>
         <global-address name='cursor' value='0x140e60058'/>
         <global-address name='selection_rect' value='0x140e60068'/>
-        <global-address name='gamemode' value='0x140e60088'/>
-        <global-address name='gametype' value='0x140e6008c'/> -4
+        <global-address name='gamemode' value='0x140e6008c'/>
+        <global-address name='gametype' value='0x140e60088'/>
```

Comment:
df::graphic is one we have the header for, so I'd be surprised if it's still incorrect. It definitely looks right in both builds on OS X and Linux.